app/src/main/res/layout/screen_enterotp.xml

```diff
@@ -0,0 +1,108 @@
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:background="@color/window_background_auth"
+  tools:ignore="Overdraw">
+
+  <LinearLayout
+    android:id="@+id/enterotp_logo_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/color_primary"
+    android:orientation="vertical"
+    android:paddingBottom="@dimen/spacing_16"
+    android:paddingTop="@dimen/spacing_4">
+
+    <android.support.v7.widget.Toolbar
+      android:id="@+id/enterotp_toolbar"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:elevation="0dp"
+      app:navigationIcon="@drawable/ic_arrow_back_24dp"
+      tools:ignore="UnusedAttribute" />
+
+    <TextView
+      android:id="@+id/enterotp_check_sms_sent_to"
+      android:layout_width="wrap_content"
+      android:layout_height="0dp"
+      android:layout_gravity="center_horizontal"
+      android:layout_marginTop="@dimen/spacing_4"
+      android:layout_weight="1"
+      android:ellipsize="end"
+      android:maxLines="1"
+      android:text="@string/enterotp_check_sms_sent_to"
+      android:textColor="@color/white_opacity_70"
+      android:textSize="@dimen/textsize_14"
+      android:textStyle="bold" />
+
+    <TextView
+      android:id="@+id/enterotp_phonenumber"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_gravity="center_horizontal"
+      android:textColor="@color/white"
+      android:textSize="@dimen/textsize_20"
+      tools:text="+911111111111" />
+  </LinearLayout>
+
+  <View
+    android:layout_width="match_parent"
+    android:layout_height="48dp"
+    android:layout_alignTop="@+id/enterotp_card"
+    android:background="@color/color_primary" />
+
+  <android.support.v7.widget.CardView
+    android:id="@+id/enterotp_card"
+    style="@style/Clinic.LoginCard"
+    android:layout_below="@+id/enterotp_logo_container"
+    android:animateLayoutChanges="true">
+
+    <ProgressBar
+      android:id="@+id/enterotp_progress"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_gravity="center"
+      android:visibility="invisible" />
+
+    <LinearLayout
+      android:id="@+id/enterotp_otp_container"
+      style="@style/Clinic.LoginCardContentContainer">
+
+      <TextView
+        style="@style/Clinic.LoginPinLabel"
+        android:labelFor="@+id/applock_pin"
+        android:text="@string/enterotp_enter_verification_code" />
+
+      <org.simple.clinic.widgets.PinEditText
+        android:id="@+id/enterotp_otp"
+        style="@style/Clinic.StaggeredEditText.Otp"
+        android:imeOptions="actionDone"
+        app:pinCharacterMask="@null"
+        android:inputType="number"
+        tools:text="123456">
```

Comment:
Will you match the text size with Zeplin later?

Reply:
I'll do it now since I need to make changes to this PR anyway.